cmake/projects/OpenSSL/externals/nasm_path_configure.cmake.in

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.2)
+
+message("################################")
+set(ENV{PATH} "@NASM_ROOT@/bin;$ENV{PATH}")
```

Comment:
Please check if `NASM_ROOT` is not empty here, if it's empty there is no need to modify `PATH`.